src/main/java/gov/cms/madie/cql_elm_translator/dto/CqlBuilderLookup.java

```diff
@@ -0,0 +1,23 @@
+package gov.cms.madie.cql_elm_translator.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Set;
+
+@Data
+@Builder
+public class CqlBuilderLookup {
+  private Set<Lookup> parameters;
+  private Set<Lookup> definitions;
+  private Set<Lookup> functions;
+  private Set<Lookup> fluentFunctions;
+
+  @Builder
```

Comment:
I missed @Data annotation on this in the initial PR. Adding it would avoid serialization issues

Reply:
@adongare added @Data